generators/entity/index.js

```diff
@@ -1566,6 +1601,12 @@ module.exports = EntityGenerator.extend({
                     this.fieldsContainManyToOne = true;
                 }
 
+                if (_.isArray(relationship.relationshipValidateRules) && relationship.relationshipValidateRules.length >= 1) {
```

Comment:
just do `if (relationship.relationshipValidateRules)` thats enough, it will give false if `relationship.relationshipValidateRules` is undefined, null or empty